target/src/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "antlr4ts",
-  "version": "0.4.0-dev",
+  "version": "0.4.1-alpha.0",
```

Comment:
Thanks @sharwell, I have no objection in principal but have been working towards some better scripting to support it.   I'll do a manual fix on this branch for now.

What's your basis for points 1 & 4?   Can you point me at some sort of best-practice write-up you suggest following?

Related: I want to eliminate files currently checked in under `target/src`, including this pesky dummy *package.json*.   I think the the package.json in the root directory should be controllingcontrolling.  That's a bigger job however, the build scripts are a bit tangled up.   I'll manually and put another commit back on this branch to restore the `-dev` for now.

`npm version` has some useful-looking subcommands to related to this, e.g. `npm version prepatch` bumps the patch-level number and includes a prerelease suffix, but...
   - it doesn't follow the `-dev` convention you mention, instead it uses sequential numbers
   - it doesn't seem to support keeping several package files in sync